Argo CD (#20)
FutureNHS/futurenhs-platform: infrastructure/README.md

   linkerd dashboard &
   ```

> 1. install Argo CD [ArgoCD](https://argoproj.github.io/argo-cd/) (insert instructions here with a script) run the `install-argo-cd.sh` script that can be found within `infrastructure/scripts` directory.

Reviewer: ```suggestion
1. To install [Argo CD](https://argoproj.github.io/argo-cd/) run the `install-argo-cd.sh` script that can be found within `infrastructure/scripts` directory.
```

   linkerd dashboard &
   ```

1. install Argo CD [ArgoCD](https://argoproj.github.io/argo-cd/) (insert instructions here with a script) run the `install-argo-cd.sh` script that can be found within `infrastructure/scripts` directory.

   ```bash
   ./install-argo-cd.sh
   ```

   Once the pods are ready, ArgoCD will be running. But it will not be accessible from outside the cluster. Use port-forward to expose a port to the service, and forward it to localhost.

   ```bash
   kubectl port-forward svc/argocd-server -n argocd 8080:443
   ```

   Your ArgoCD cluster will be available at https://localhost:8080. Leave the port-forward running in the foreground of your terminal. Open another terminal window.

   In the UI, you will not be able to log in yet.

   The initial password is autogenerated to be the pod name of the Argo CD API server. This can be retrieved with the command:

   ```bash
   kubectl get pods -n argocd -l app.kubernetes.io/name=argocd-server -o name | cut -d'/' -f 2
   ```

   Using the username admin and the password from above, login to Argo via localhost

   ```bash

>    argocd login https://localhost:8080

Reviewer 2: `argocd` sounds like something you needed to install on your mac. Probably want instructions on how to do this?

Author: yes, indeed. will update docs

Author: added into the script. hope i did it right
